### PR TITLE
Handle time stamps from containerd checkpoint archives

### PR DESCRIPTION
**Pull request by adrianreber**
This fixes containerd archive created time handling and adds checkpointed time for all archives.

### CODE REVIEW

**Comment by rst0git** [comment]
@adrianreber Would it be possible to also update the examples in [README.md](https://github.com/checkpoint-restore/checkpointctl/blob/main/README.md)?

**Comment by adrianreber** [comment]
> @adrianreber Would it be possible to also update the examples in [README.md](https://github.com/checkpoint-restore/checkpointctl/blob/main/README.md)?

Sure. Done.

File: internal/json.go [hunk 1]
```diff
@@ -112,6 +114,7 @@ func RenderJSONView(tasks []Task) error {
 			ID:            info.configDump.ID,
 			Runtime:       info.configDump.OCIRuntime,
 			Created:       info.containerInfo.Created,
+			Checkpointed:  info.configDump.CheckpointedAt.Format(time.RFC3339),
```
Comment (rst0git) [inline internal/json.go:117]: It looks like Podman currently does not set `CheckpointedAt`

Comment (adrianreber) [reply internal/json.go:117]: Interesting. We should set that. I will change this PR to skip it if empty.

Comment (adrianreber) [reply internal/json.go]: Removed the zero value if not set as in the Podman case.

**Review by rst0git — APPROVED** [review]
LGTM

**Pull request merged**